jobs/service-fabrik-apiserver/spec

```diff
@@ -2,28 +2,46 @@
 name: service-fabrik-apiserver
 
 packages:
-  - service-fabrik-apiserver
+- apiserver
 
 templates:
-  bin/service-fabrik-apiserver.erb: bin/service-fabrik-apiserver
+  bin/ensure_apiserver_healthy.erb: bin/ensure_apiserver_healthy
+  bin/post-start.erb: bin/post-start
   config/bpm.yml.erb: config/bpm.yml
   config/etcd-ca.crt.erb: config/etcd-ca.crt
   config/etcd.crt.erb: config/etcd.crt
   config/etcd.key.erb: config/etcd.key
-
+  config/apiserver-key.pem.erb: config/apiserver-key.pem
```

Comment:
Can we have name like **apiserver.key , apiserver.crt** in order to keep consistent across system?

Reply:
ok